velox/functions/prestosql/BinaryFunctions.h

```diff
@@ -403,4 +403,20 @@ struct ToBigEndian64 {
   }
 };
 
+template <typename T>
+struct ToIEEE754Bits64 {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(
+      out_type<Varbinary>& result,
+      const arg_type<double>& input) {
+    static constexpr auto kResultLength = 8 * sizeof(uint64_t);
+    result.resize(kResultLength);
+    std::bitset<64> bits =
+        std::bitset<64>(*reinterpret_cast<const unsigned long long*>(&input));
+    std::string buffer = bits.to_string();
```

Comment:
@Yuhta Jimmy, I can have the buffer preallocated by doing something like `std::string buffer(64, '\0');` however, `bits::toString((const uint64_t*)&input, 0, 64)` will not get me the desired bits representation (as I am converting `double` to `const unsigned long long` (aka `const uint64_t`) pointer through `reinterpret_cast` and then using the uint64_t value for binary rep.). With what you suggested, all my testcases are failing (was passing earlier). If you could clarify further that would be great help. Thanks!

Reply:
I see the endianness is flipped.  Can you try call `std::reverse` on the buffer after `toString`?

Reply:
Actually we will almost for sure need `void toString(const uint64_t* bits, int offset, int size, char* out)` as RVO won't kick in to avoid the allocation of the new `string`.